projects/backend/fronts.ts

```diff
@@ -89,9 +112,17 @@ export const getFront = async (
 
     const config: FrontsConfigResponse = (await resp.json()) as FrontsConfigResponse
     if (!(id in config.fronts)) throw new Error('Front not found')
-    const front = config.fronts[id]
+    const front = {
+        ...config.fronts[id],
+        collections: (await Promise.all(
```

Comment:
@AWare will this have any knock on effect for you on what you've been working on?

Reply:
Probably will review tomorrow.

Reply:
This will need a fair amount of rework on the archiver. And personally would prefer this kind of change happens and after Tuesdays meeting.

Reply:
I had a think about this overnight and I think overall this is the way to go shape-wise as there's no scenario where the app will not immediately ask the backend/zip file for every collection right after fetching a front, so we might as well avoid the round trip?

Reply:
Doing the fetching in the app requires a bit of a cleanup in the fetching code (less nested hooks, more promises) which is something we need to do anyway so I'm happy to do that instead of changing the backend – but even after doing that I still think the backend bringing every collection with each front is a cleaner way to go!

Reply:
That would be amazing. I think we can revisit the assumptions from the beginning about the API as we've learnt a lot since then. I'm not even sure we shouldn't ship a single JSON response.

Reply:
Let's chat in person at standup because I'm not sure today right before launch is the right day to rewrite data fetching